docs/content.zh/docs/deployment/resource-providers/standalone/kubernetes.md

```diff
@@ -25,61 +25,65 @@ specific language governing permissions and limitations
 under the License.
 -->
 
```

Comment:
Maybe here need to add the link tag ?

Reply:
ok, take your advice. It has been fixed.